source/shared/cpp/ObjectModel/Carousel.h

```diff
@@ -38,15 +40,19 @@ class Carousel : public CollectionCoreElement
     std::optional<bool> GetAutoLoop() const;
     void setAutoLoop(const std::optional<bool>& value);
 
+    std::optional<bool> GetRtl() const;
+    void SetRtl(const std::optional<bool>& value);
+
 private:
     void PopulateKnownPropertiesSet();
 
     std::vector<std::shared_ptr<AdaptiveCards::CarouselPage>> m_pages;
-    std::string m_heightInPixels;
+    unsigned int m_heightInPixels{0};
```

Comment:
Does this make the default height 0px?

Reply:
the value will be read from the shared model. It's a sanity check and makes compiler happy. if it's zero, the value won't be used.